lib/tasks/index.rake

```diff
@@ -46,4 +47,17 @@ namespace :index do
     communities = DataspaceCommunities.new
     File.write(cache_file, JSON.pretty_generate(communities.tree))
   end
+
+  desc 'Prints to console the current Solr URLs and how they are configured'
+  task print_solr_urls: :environment do
+    puts "Solr alias.: #{SolrCloudHelper.alias_url}"
+    puts "Solr reader: #{SolrCloudHelper.collection_reader_url}"
+    puts "Solr writer: #{SolrCloudHelper.collection_writer_url}"
+  end
+
+  desc 'Updates the Solr alias to point to the most up to date collection'
+  task update_solr_alias: :environment do
+    puts "Solr updated: #{SolrCloudHelper.update_solr_alias!}"
+    Rake::Task['index:print_solr_urls'].invoke
+  end
```

Comment:
These two tasks are for debugging and troubleshooting, not that I expect us to use them 🤣